### ISSUE TITLE
easyblocks shouldn't be defining $FPATH (imkl, acml, inspector, vtune)

### ISSUE THREAD

**Issue by boegel** [hpcugent/easybuild-easyblocks#820]
`$FPATH` is not what we think it may be... It's _not_ the equivalent of `$CPATH` for Fortran.

And it screws up tab completion in zsh.

cc @rtmclay

**Comment by boegel** [comment]
Currently, these easyblocks define `$FPATH` (the EasyBuild framework itself doesn't):

```
$ grep -RI FPATH easybuild
easybuild/easyblocks/a/acml.py:        for key in ['CPATH', 'FPATH']:
easybuild/easyblocks/i/imkl.py:                    'FPATH': ['mkl/include', 'mkl/include/fftw'],
easybuild/easyblocks/i/imkl.py:                    'FPATH': ['include']
easybuild/easyblocks/i/imkl.py:                    'FPATH': ['include'],
easybuild/easyblocks/i/inspector.py:            'FPATH': [os.path.join(self.subdir, 'include')],
easybuild/easyblocks/v/vtune.py:            'FPATH': [os.path.join(self.subdir, 'include')],
```

**Comment by boegel** [comment]
fixed in #1140
